noise/README.md

```diff
@@ -169,15 +165,13 @@ identity keypair to use Noise, noise-libp2p uses a separate static keypair for
 Noise that is distinct from the peer's identity keypair.
 
 A given libp2p peer will have one or more static Noise keypairs throughout its
-lifetime. Implementations MAY allow persisting static Noise keys across process
-restarts, or they may generate new static Noise keys when initializing the
-noise-libp2p module. 
-
-Systems which enable the [Noise Pipes
-pattern](#optimistic-0-rtt-with-noise-pipes) are likely to benefit from a longer
-lifetime for static Noise keys, as the static key is used in the optimistic
-case. Other systems may prefer to cycle static Noise keys frequently to reduce
-exposure.
+lifetime. Because the static key is authenticated using the libp2p identity key,
+it is not necessary for the key to actually be "static" in the traditional
+sense, and implementations MAY generate a new static Noise keypair for each new
+session. Alternatively, a single static keypair may be generated when
+noise-libp2p is initialized and used for all sessions. Implementations SHOULD
+NOT store the static Noise key to disk, as there is no benefit and a hightened
+risk of exposure.
```

Comment:
Reversing the suggestion makes sense now, as the handshake that would've benefitted from a long-lived/persisted key is now gone (the IK handshake).